clang/test/CAS/remote-cache-service.c

```diff
@@ -1,16 +1,16 @@
 // REQUIRES: remote-cache-service, shell
 
 // Need a short path for the unix domain socket.
-// RUN: CACHE=%{remote-cache-dir}/$(basename %t)
-// RUN: rm -rf $CACHE && mkdir -p $CACHE
+// RUN: SOCKET_PATH=%{remote-cache-dir}/$(basename %t)
```

Comment:
shouldn't we configure `%{remote-cache-dir}` just be `%{socket-path}` to avoid this env?

Reply:
The intention is to form a socket path that is unique to the particular test file, so that there is no issue if multiple concurrent test files are using `llvm-remote-cache-test`. 
Ideally `lit` would provide a `%{socket-path}` that is specific for each test file but it's not clear to me whether it is possible to define such a thing or not.

Reply:
I see. Also we have `%basename_t` that can use directly. using env variable is kind of unnecessary and enforce shell requirement. I guess the use of `clang-cache` make it harder to.

Reply:
Oh, I wasn't aware of `%basename_t`! Since I can avoid calling `basename` I might as well drop the environment variable as well and repeat the path.

Reply:
See updated commit.